images/arch/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/archlinux/base:latest
+MAINTAINER Erazem Kokot <contact at erazem dot eu>
+
+ENV NAME=arch-toolbox VERSION=rolling
+LABEL com.github.containers.toolbox="true" \
+      com.github.debarshiray.toolbox="true" \
+      name="$FCG/$NAME" \
+      version="$VERSION" \
+      usage="This image is meant to be used with the toolbox command" \
+      summary="Base image for creating Arch toolbox containers" \
+      maintainer="Erazem Kokot <contact at erazem dot eu>"
+
+RUN pacman -Syu
+
+# Install packages from the extra-packages file
+COPY extra-packages /
+RUN pacman -Sy --noconfirm $(<extra-packages)
+RUN rm /extra-packages
+
+# Clean up all local caches
+RUN pacman -Scc
+
+CMD /bin/sh
```

Comment:
I tried to build the image and when I got to this point, it errored out with `no such command`. That'll probably need some investigation??

Reply:
I've tested pacman -Scc and it works fine, did you mean no such command for /bin/sh or pacman?

Reply:
For `CMD /bin/sh`.

Reply:
Ok, I'll check it out (I'm pretty sure it was because `bash` is a part of the `base` metapackage now and I haven't added it yet when you were testing the image).